cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractAssemblyOperator.java

```diff
@@ -253,24 +253,25 @@ public final CountDownLatch reconcileAll(String trigger, String namespace) {
             future -> {
                 Watch watch = resourceOperator.watch(namespace, new Watcher<T>() {
                     @Override
-                    public void eventReceived(Action action, T cm) {
-                        String name = cm.getMetadata().getName();
+                    public void eventReceived(Action action, T cr) {
+                        String name = cr.getMetadata().getName();
+                        String actualNamespace = cr.getMetadata().getNamespace();
```

Comment:
If we're going to have two namespace variables floating around it might make sense to rename (or at least better document) the existing `namespace` since it's potentially different from the `actualNamespace` now.